### ISSUE TITLE
Eating text before dashes

### ISSUE THREAD

**Issue by deathau** [lynchjames/obsidian-day-planner#43]
I've started using day planner extensively, but come up against an annoying issue: It will randomly eat (delete) text before a dash on a line.

For example, if I type:
```markdown
this is a test - there is text before the hyphen
```
Then after some seconds text is removed and I'm left with
```markdown
 - there is text before the hyphen
```
Ctrl+Z brings the text back, but it disappears again shortly. The only ways around this I've found are to disable the plugin or use em-dash instead of dash.

This only happens on day plan linked notes.

**Comment by Kr-Fr** [comment]
I have the same problem...

**Comment by lynchjames** [comment]
This should now be fixed in [0.5.7](https://github.com/lynchjames/obsidian-day-planner/releases/tag/0.5.7).
